packages/common-ui-web/src/components/VehiclePartSelection.tsx

```diff
@@ -0,0 +1,93 @@
+import { PartSelectionOrientation, VehiclePart, VehicleType } from '@monkvision/types';
+import { useEffect, useState } from 'react';
+import { useMonkTheme } from '@monkvision/common';
+import { Icon } from '../icons';
+import { VehicleDynamicWireframe, VehicleDynamicWireframeProps } from './VehicleDynamicWireframe';
+
+export interface VehiclePartSelectionProps {
+  vehicleType: VehicleType;
+  orientation?: PartSelectionOrientation;
+  /**
+   * emit when change in the selected parts.
+   */
+  onPartsSelected?: (parts: VehiclePart[]) => void;
+}
```

Comment:
Add TSDoc for this interface and its properties. Remember to add capital letters at the start of sentences and dots at the end.